### PR TITLE
Adds documentation for Plaato Airlock

### PR DESCRIPTION
**Pull request by JohNan**
**Description:**
Adds a new integration for the Plaato Airlock. Plaato Airlock is a tool for beer brewers that wants a unique insight in the fermentation process. It will create sensors for the data that is pushed to a webhook from the airlock.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23727

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by JohNan** [comment]
I just got a green light from Plaato to use their logo in the documentation. Please wait with merge until I have updated the PR

File: source/_components/plaato.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+description: "Instructions on how to integrate Plaato Airlock sensors within Home Assistant."
+date: 2019-05-07 01:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: plaato.png
+ha_release: "0.93"
+ha_category: Sensor
+ha_iot_class: "Cloud Push"
```
Comment (klaasnicolaas) [inline source/_components/plaato.markdown]:
```suggestion
ha_iot_class: Cloud Push
```

File: source/_components/plaato.markdown [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+layout: page
+title: "Plaato Airlock"
+description: "Instructions on how to integrate Plaato Airlock sensors within Home Assistant."
+date: 2019-05-07 01:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: plaato.png
+ha_release: "0.93"
```
Comment (klaasnicolaas) [inline source/_components/plaato.markdown]:
```suggestion
ha_release: 0.93
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by JohNan** [comment]
@klaasnicolaas The example at https://developers.home-assistant.io/docs/en/documentation_create_page.html says those values should use quotes. Maybe the docs need to be updated?

Pushed the changes you suggested.

**Comment by klaasnicolaas** [comment]
In the case of the release version, it should only be for versions that end with a 0, such as 0.30, 0.40, 0.50 etc. For the iot class it is wrong in the example.

**Comment by JohNan** [comment]
@klaasnicolaas I understand. Is the changes fine now then?

**Review by klaasnicolaas — DISMISSED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @JohNan! 👍

**Pull request merged**